packages/typescript/ai/tests/debug-logging-chat.test.ts

```
@@ -0,0 +1,280 @@
+import { describe, expect, it, vi } from 'vitest'
+import { chat } from '../src/activities/chat/index'
+import type { Logger } from '../src/logger/types'
+import type { StreamChunk } from '../src/types'
+import { collectChunks, createMockAdapter, ev } from './test-utils'
```

ESLint

[error] `./test-utils` import should occur before type import of `../src/logger/types`

(import/order)